src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -663,7 +663,14 @@ public string LiteralPath
             get => _path;
             set
             {
-                _path = value;
+                var paths = SessionState.Path.GetResolvedProviderPathFromPSPath(value, out _);
+                var hasWildcard = WildcardPattern.ContainsWildcardCharacters(value);
+                if (hasWildcard || paths.Count > 1)
```

Comment:
The point of `-LiteralPath` is to allow wildcard chars in the path, right? But you are rejecting inputs with wildcards here.

Reply:
I... _think_ you might have that backwards? 😄

Reply:
`LiteralPath` accepts wildcards, it just doesn't expand/evaluate them

Reply:
_Oooooh_. In that case, I don't think the `GetResolvedProviderPath` method(s) can be used here. Is there an alternative equivalent for provider paths that treat the path as literal @daxian-dbw?